types/email-templates/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "juice": "^7.0.0"
+    }
```

Comment:
It seems that this particular change caused the project build failure described earlier: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/49748/files#r606746312.

Reply:
Hey, the change was redacted to remove terse integration:
#52180